sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/BeamZetaSqlCalcRel.java

```diff
@@ -169,12 +169,10 @@ private static TimestampedFuture create(Instant t, Future<Value> f) {
     private final Schema outputSchema;
     private final String defaultTimezone;
     private final boolean verifyRowValues;
+    private transient List<Integer> referencedColumns = ImmutableList.of();
```

Comment:
Findbugs is unhappy with these two lines. Tag them with `@SuppressFBWarnings("SE_TRANSIENT_FIELD_NOT_RESTORED")`

Also need: `import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;`

Reply:
fixed.